lib/matplotlib/axes/_axes.py

```diff
@@ -5843,6 +5846,8 @@ def hist(self, x, bins=10, range=None, normed=False, weights=None,
                     xvals.append(x.copy())
                     yvals.append(y.copy())
 
+            fill_kwargs = dict() if not input_empty else dict(linewidth=0)
```

Comment:
Just making sure, is autoscaling in the x still working even with
zero-height elements on the edges?

On Fri, Mar 13, 2015 at 7:31 PM, Thomas A Caswell <notifications@github.com>
wrote:

> In lib/matplotlib/axes/_axes.py
> <https://github.com/matplotlib/matplotlib/pull/4119#discussion_r26430425>:
>
> > @@ -5843,6 +5846,8 @@ def hist(self, x, bins=10, range=None, normed=False, weights=None,
> >                      xvals.append(x.copy())
> >                      yvals.append(y.copy())
> >
> > +            fill_kwargs = dict() if not input_empty else dict(linewidth=0)
>
> Again. I think showing the line is the correct thing to do. My main
> motivation is that if some one now wants to update the histogram artists,
> this will cause a great deal of trouble.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4119/files#r26430425>.
>